data/json/items/ammo/460.json

```diff
@@ -1,10 +1,10 @@
 [
   {
-    "id": "460fmj",
```

Comment:
changing id will affect savegames very badly, have you considered that?

Reply:
Yes.  Kevin advised me not to bother migrating if I pushed a PR quickly enough.

Reply:
This id was added earlier tonight, it's not worthwhile to add migration code for a tiny window like that.